src/components/directory-content.jsx

```diff
@@ -1110,17 +696,10 @@ const DirectoryContent = () => {
 
     return (
         <>
-            <div
-                style={{
-                    display: 'flex',
-                    flexDirection: 'column',
-                    flexGrow: 1,
-                }}
-                onContextMenu={(e) => onContextMenu(e)}
-            >
+            {rows && <ContentToolbar selectedElements={checkedRows} />}
```

Comment:
include this toolbar in below grid so we can use right click here just like in main branch ?

Reply:
Moving it into the Grid breaks everything down and I wasn't able to find another workaround as this one

Reply:
since it changes the behaviour, we should ask the PO if this change is OK
we could add a box wrapping both which would hold the "onContextMenu" handler